tests/page.js

```diff
@@ -175,9 +175,24 @@ module.exports = function Page(browser) {
 		});
 	};
 
+	this.waitForVisibleElement = function (getElementFn, timeout) {
+		var foundVisibleElement;
+		timeout = timeout || 500;
```

Comment:
The magic 500 is some kind of a hack at this place. I experienced slow tests when running the clear-complete-button tests against ```backbone``` framework. With Backbone approach the items gets hidden with the filter buttons whereas my Aurelia solution rebuilds the whole items list. Thus, for the backbone variant the ```waitForVisibleElement()``` helper will wait for hidden items to get visible in some cases.
:exclamation: Needs further investigation.